lib/ts/recipe/multifactorauth/types.ts

```diff
@@ -154,6 +154,7 @@ export type GetPhoneNumbersForFactorsFromOtherRecipesFunc = (
 
 export const FactorIds = {
     EMAILPASSWORD: "emailpassword",
+    WEBAUTHN: "webauthn",
```

Comment:
You need to modufy the user object.. For example, we are adding pass keys credentials for them. So now login methods need to have passkeys credentials ids? And the root of the user object needs to also have a passkeys credentials ids array? What else needs to be added?

Reply:
Yeah. So after discussing went over the code and schema again and decided to add a webauthn object containing the credential ids. Besides this, i've also added the relying party id to the credentials schema (see here: https://docs.google.com/document/d/1G7tO9_dSNi8wur3ajGg4pq-wiHatKDbHv2sBt-uSbQg/edit#heading=h.td4zxgo1675k) because we don't have a way of knowing what credentials are available for a relying party id. 

Also added some more info for the CDI docs to include the webauthn info in the user object (see here: https://docs.google.com/document/d/1G7tO9_dSNi8wur3ajGg4pq-wiHatKDbHv2sBt-uSbQg/edit#heading=h.r1r1xaejv0ga).

Done